templates/tfengine/components/cicd/terraform.tfvars

```diff
@@ -12,6 +12,45 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+build_editors   = [
+  {{- if has . "build_editors"}}
+  {{- range .build_editors}}
+  "{{.}}",
+  {{- end}}
+  {{- end}}
+]
```

Comment:
This is kind of hard to read due to a lot of templating. I don't really see an easy (and more readable) way to set it to an empty list if not present but filled out others.

However, this construction suggests to me that we're implying a default value if this variable wasn't set in the recipe. I think this can be done by setting a `default = []` in variables.tf and using `{{hclField . "build_editors"}}` here.

Reply:
OMG I cannot believe how good it worked haha

Maybe you'd like to check the next vars. I reorganized the triggers into submodules and the `triggers` variable is too verbose. Maybe we can simplify it but since it's a nested data structure, it may be more difficult